app/user/rest.py

```diff
@@ -37,15 +42,8 @@ def update_user(user_id):
         delete_model_user(user)
     else:
         status_code = 200
-        # TODO there has got to be a better way to do the next three lines
-        update_user, errors = user_schema.load(request.get_json())
-        if errors:
-            return jsonify(result="error", message=errors), 400
-        update_dict, errors = user_schema.dump(update_user)
-        # TODO FIX ME
-        # Remove update_service model which is added to db.session
         db.session.rollback()
-        save_model_user(user, update_dict=update_dict)
+        save_model_user(user, update_dict=request.get_json())
```

Comment:
Yes but that causes exceptions with non nullable fields on model if load is called. There will need to be another way of doing that rather than calling load to get validation.